### PR TITLE
MGMT-11274: assisted: add nutanix test to assisted-service

### PR DESCRIPTION
**Pull request by vrutkovs**
This also includes a periodic job to check nutanix setup. Follow up for #31726

### CODE REVIEW

File: ci-operator/jobs/openshift/assisted-service/openshift-assisted-service-master-periodics.yaml [hunk 1]
```diff
@@ -890,6 +890,79 @@ periodics:
...
+  interval: 48h
+  labels:
+    ci-operator.openshift.io/cloud: nutanix
+    ci-operator.openshift.io/cloud-cluster-profile: nutanix
+    ci-operator.openshift.io/cluster: build01
+    ci-operator.openshift.io/variant: edge
+    ci.openshift.io/generator: prowgen
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-assisted-service-master-edge-e2e-nutanix-assisted-periodic
+  spec:
```
Comment (osherdp) [inline ci-operator/jobs/openshift/assisted-service/openshift-assisted-service-master-periodics.yaml:909]:
```suggestion
  reporter_config:
    slack:
      channel: '#assisted-deployment-ci'
      job_states_to_report:
      - failure
      - error
      report_template: '{{ if eq .Status.State "success" }} :green_jenkins_circle:
        {{ else }} :red_jenkins_circle: {{ end }} Job *{{.Spec.Job}}* ended with *{{.Status.State}}*.
        <{{.Status.URL}}|View logs>'
  spec:
```
so we can be aware of failures if they occur

File: ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml [hunk 2]
```diff
@@ -991,6 +991,30 @@ tests:
...
+- as: e2e-nutanix-assisted
+  cluster: build01
+  optional: true
+  steps:
+    cluster_profile: nutanix
+    env:
+      ASSISTED_CONFIG: |
+        NUM_WORKERS=0
+        MAKEFILE_TARGET=test
+        DISABLED_HOST_VALIDATIONS=belongs-to-majority-group
```
Comment (osherdp) [inline ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml:1003]: maybe we should do the same for the job against test-infra?

File: ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml [hunk 3]
```diff
@@ -991,6 +991,30 @@ tests:
...
+      PLATFORM: nutanix
+    workflow: assisted-nutanix
+- as: e2e-nutanix-assisted-periodic
```
Comment (osherdp) [inline ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml:1006]: out of convention all of the test-infra based periodic jobs are against assisted-test-infra repo

**Review by osherdp — CHANGES_REQUESTED** [review]

File: ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml (hunk 2 above)
Comment (vrutkovs) [reply ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml:1003]: Surprisingly, test-infra job didn't need it. I can't remember how this was fixed there. Leaving this for a follow up fix

**Review by osherdp — APPROVED** [review]

**Pull request merged**